features/send-money-to-receiver.feature

```diff
@@ -3,6 +3,7 @@ Feature: Send Money to Receiver
   Background:
     Given I have selected English language
     And I have accepted "Terms of use"
+    And I agree to send logs to remote server
```

Comment:
I guess the duplication of the background stories here is a good indicator that we need to refactor this into a unified step that does all necessary background work to get to the meat of other tests.

Added a card for this: https://issues.serokell.io/issue/DAEF-365